lib/chef/knife/bootstrap/common_bootstrap_options.rb

```diff
@@ -96,6 +96,11 @@ def self.included(includer)
               :default => false,
               :description => "Optional. It shows chef convergence logs in detail."
 
+            option :chef_service_interval,
+              :long => "--chef-service-interval INTERVAL",
+              :description => "Optional. It specifies the frequency (in minutes) at which the chef-service runs.
+                                Pass 0 if you don't want the chef-service to be installed on the target machine."
+
```

Comment:
```
$ git grep BootstrapOptions
lib/chef/knife/azure_server_create.rb:      include Knife::Bootstrap::BootstrapOptions
lib/chef/knife/azurerm_server_create.rb:      include Knife::Bootstrap::CommonBootstrapOptions
lib/chef/knife/bootstrap/bootstrap_options.rb:      module BootstrapOptions
lib/chef/knife/bootstrap/common_bootstrap_options.rb:      module CommonBootstrapOptions
lib/chef/knife/bootstrap_azure.rb:      include Knife::Bootstrap::CommonBootstrapOptions
lib/chef/knife/bootstrap_azurerm.rb:      include Knife::Bootstrap::CommonBootstrapOptions
```

It seems like `BootstrapOptions` vs `CommonBootstrapOptions` has gotten out of hand. Although I realize that these options are slightly different as cloud-api appears to optional for ASM but not ARM. So maybe nothing to do here, but this "smells."

Can you at least add comments to the top of BootstrapOptions and CommonBootstrapOptions to describe how they're supposed to be different?